translations/es.json

```diff
@@ -1,6 +1,6 @@
 {
   "error-occured": "Un error ha impedido que la extensión funcione apropiadamente: ",
-  "version-update-message": "Ahora estás usando ezpp! v{0}",
+  "version-update-message": "¡Ahora estás usando ezpp! v{0}",
```

Comment:
I'm not sure if this is correct, the usage of this is the update message: `Ahora estás usando ezpp! v1.2.3`
This doesn't end in `!` so I guess it shouldn't begin with `¡`?

Reply:
Is the exclamation mark part of the name? as in `ezpp!`? If so then yeah `¡` should be removed.

If not part of the name then these both are correct:
`¡Ahora estás usando ezpp! v1.2.3`
`¡Ahora estás usando ezpp v1.2.3!`
(The latter seems better tho)

But it's also acceptable (but not quite right) not to use `¡`. I mean, most of Spanish speaking people don't care about this tbh, we don't use it in everyday life cuz we're lazy.

So I guess it doesn't really matter.

Reply:
So, you want me to remove `¡`? I mean it's up to you, it's acceptable not to use `¡`

Reply:
yes, the ! is part of the name.

Reply:
ok, reverted.